src/lib/Containers/index.tsx

```diff
@@ -14,12 +15,13 @@ export default {
   Artist,
   BidFlow,
   Conversation,
-  Home,
   Gene,
-  Sale,
-  RegistrationFlow,
-  WorksForYou,
-  MyProfile,
+  Fair,
```

Comment:
Separately from the `export * from` bit we attempted, why are we re-exporting `Scenes` components from this `Containers` file? It looks like `AppRegistry` also imports some scenes directly, is there a difference? https://github.com/artsy/emission/blob/3cf7d31e085051d964a85890d18aec2ebc3b72a3/src/lib/AppRegistry.tsx#L21

Reply:
I think that might be the autocomplete, I think we just thought that exporting everything from one place might make it easier to work with (for example we could write a test like this)